### PR TITLE
test: initial commit with tests

### PR DESCRIPTION
**Pull request by pcheremu**
# What ❔

Tests for:
- output-dir
- zkasm

+ TMP dir for new test
+ extended output file name detection

## Checklist

- [ ] PR title corresponds to the body of PR.
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `cargo fmt` and checked with `cargo clippy`.

### CODE REVIEW

**Comment by pcheremu** [comment]
@antonbaliasnikov hello, do we also need to change the workflow for zkvyper?
`use-prebuilt-zkvyper: ${{ github.event.pull_request.draft || 'true' }}`

**Comment by hedgar2017** [comment]
> @antonbaliasnikov hello, do we also need to change the workflow for zkvyper? `use-prebuilt-zkvyper: ${{ github.event.pull_request.draft || 'true' }}`

@antonbaliasnikov please sync!

**Comment by antonbaliasnikov** [comment]
> @antonbaliasnikov hello, do we also need to change the workflow for zkvyper? `use-prebuilt-zkvyper: ${{ github.event.pull_request.draft || 'true' }}`

Yes, thanks. I fixed that.

File: cli-tests/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "test": "npx jest --verbose --testPathPattern=common.test.ts"
+    "test": "npx jest --verbose --testPathPattern="
```
Comment (antonbaliasnikov) [inline cli-tests/package.json:10]: Maybe, the whole `--testPathPattern=` could be removed in this case?

**Review by antonbaliasnikov — APPROVED** [review]
LGTM, please, check one comment about empty pattern.

File: cli-tests/package.json (hunk 1 above)
Comment (pcheremu) [reply cli-tests/package.json:10]: We definitely could remove it. I keep it on purpose to simplify test run using a specific test file. So to do this you just have to copy this string or insert any file test that you want to run

Comment (pcheremu) [reply cli-tests/package.json:10]: Was discussed in DM to keep it as is.

**Pull request merged**